client-app/pages/account/order-details.vue

```diff
@@ -1,93 +1,152 @@
-                <Vendor v-if="group.vendor" :vendor="group.vendor" class="inline-flex flex-row items-end gap-x-3" />
-                <span v-else class="text-gray-400" v-t="`pages.account.order_details.empty_vendor_label`" />
-              </div>
-
-              <OrderLineItems :items="group.items" class="lg:rounded" />
-            </div>
-          </template>
+    <VcLayoutWithRightSidebar is-sidebar-sticky>
+      <!-- Order Data Widget -->
+      <VcCardWidget v-if="isMobile" :title="$t('pages.account.order_details.order_data_card.title')" hideMobileTitle>
+        <div class="flex flex-col space-y-1.5 text-15">
+          <p v-if="order.createdDate">
+            <span class="font-extrabold">
+              {{ $t("pages.account.order_details.order_data_card.creates_label") }}
+            </span>
+            {{ $d(order.createdDate, "long") }}
+          </p>
+          <p class="flex items-center" v-if="order.status">
+            <span class="font-extrabold mr-2">
+              {{ $t("pages.account.order_details.order_data_card.status_label") }}
+            </span>
+            <TableStatusBadge :status="order.status" />
+          </p>
         </div>
-
-        <!-- Items not grouped by Vendor -->
-        <div v-else class="bg-white lg:mb-6 lg:rounded -mx-5 md:mx-0 lg:shadow-md-x lg:pt-5 lg:px-7">
-          <div class="bg-white shadow-light-lg mb-4 p-7 lg:mb-0 lg:pb-5 lg:px-0 lg:pt-0 lg:rounded lg:shadow-none">
-            <OrderLineItems :items="orderItems" class="lg:rounded" />
+      </VcCardWidget>
+
+      <!-- Items grouped by Vendor -->
+      <div
+        v-if="$cfg.line_items_group_by_vendor_enabled"
+        class="bg-white md:mx-0 lg:mb-6 lg:rounded lg:shadow-md-x lg:pt-5 lg:px-7"
+      >
+        <template v-for="(group, vendorId) in orderItemsGroupedByVendor" :key="vendorId">
+          <div
+            v-if="group.items.length"
+            class="bg-white shadow-light-lg mb-4 px-7 pt-4 lg:mb-0 lg:pb-5 lg:px-0 lg:pt-0 lg:rounded lg:shadow-none"
+          >
+            <!-- Vendor -->
+            <VcVendor v-if="group.vendor" :vendor="group.vendor" class="pb-3"></VcVendor>
+
+            <OrderLineItems :items="group.items" class="lg:rounded" />
           </div>
+        </template>
+      </div>
+
+      <!-- Items not grouped by Vendor -->
+      <div v-else class="bg-white lg:mb-6 lg:rounded -mx-5 md:mx-0 lg:shadow-md-x lg:pt-5 lg:px-7">
+        <div class="bg-white shadow-light-lg mb-4 p-7 lg:mb-0 lg:pb-5 lg:px-0 lg:pt-0 lg:rounded lg:shadow-none">
+          <OrderLineItems :items="orderItems" class="lg:rounded" />
         </div>
+      </div>
 
-        <!-- Gifts section -->
-        <AcceptedGifts :items="giftItems" />
+      <!-- Gifts section -->
+      <AcceptedGifts :items="giftItems" />
+
+      <!-- Order comment section -->
+      <VcSection v-if="order?.comment && !isMobile" class="shadow-inner mt-5 p-6 lg:shadow" content-classes="pt-4">
```

Comment:
use `<VcSectionWidget />` component.
VcSection will be removed from project soon.